src/sidebar/components/annotation-header.js

```diff
@@ -12,10 +12,27 @@ import Button from './button';
 import SvgIcon from '../../shared/components/svg-icon';
 import Timestamp from './timestamp';
 
+/**
+ * @typedef {import("../../types/api").Annotation} Annotation
+ */
+
+/**
+ * @typedef AnnotationHeaderProps
+ * @prop {Annotation} annotation - The annotation in question
```

Comment:
```suggestion
 * @prop {Annotation} annotation
```

I don't think the property description provides any useful info here.